ci/scripts/jenkins_build.sh

```diff
@@ -0,0 +1,134 @@
+      mkdir -p ${CONDA_BLD_DIR}
+      # The --no-build-id bit is needed for sccache
+      CONDA_ARGS="--no-build-id --output-folder ${CONDA_BLD_DIR} --skip-existing --no-test" ${MORPHEUS_ROOT}/ci/conda/recipes/run_conda_build.sh libcudf cudf
+
+      gpuci_logger "sccache usage for cudf build:"
+      sccache --show-stats
+      ZS=$(sccache --zero-stats)
+
+      gpuci_logger "Archiving cuDF build"
+      cd $(dirname ${CONDA_BLD_DIR})
+      tar cfj ${CUDF_CONDA_TAR} $(basename ${CONDA_BLD_DIR})
+      cd -
+      aws s3 cp --no-progress ${CUDF_CONDA_TAR} ${CUDF_CONDA_CACHE_URL}
+else
+      gpuci_logger "Cache hit, using cached cuDF"
+      cd $(dirname ${CONDA_BLD_DIR})
+      tar xf ${CUDF_CONDA_TAR}
+      cd -
+fi
+
+gpuci_logger "Installing cuDF"
+mamba install -q -y -c file://${CONDA_BLD_DIR} -c nvidia -c rapidsai -c conda-forge libcudf cudf
+
+gpuci_logger "Installing other dependencies"
+mamba env update -q -n morpheus -f ./docker/conda/environments/cuda${CUDA_VER}_dev.yml
+
+gpuci_logger "Check cmake & ninja"
+cmake --version
+ninja --version
+
+gpuci_logger "Configuring cmake for Morpheus"
+cmake -B build -G Ninja \
+      -DCMAKE_MESSAGE_CONTEXT_SHOW=ON \
+      -DMORPHEUS_BUILD_BENCHMARKS=ON \
+      -DMORPHEUS_BUILD_EXAMPLES=ON \
+      -DMORPHEUS_BUILD_TESTS=ON \
+      -DMORPHEUS_USE_CONDA=ON \
+      -DMORPHEUS_PYTHON_INPLACE_BUILD=ON \
+      -DMORPHEUS_USE_CCACHE=OFF \
+      -DCMAKE_C_COMPILER_LAUNCHER=sccache \
+      -DCMAKE_CXX_COMPILER_LAUNCHER=sccache \
+      -DCMAKE_CUDA_COMPILER_LAUNCHER=sccache \
+      .
+
+gpuci_logger "Building Morpheus"
+cmake --build build -j --parallel ${PARALLEL_LEVEL}
+
+gpuci_logger "sccache usage for morpheus build:"
+sccache --show-stats
+
+gpuci_logger "Installing Morpheus"
+pip install -e ${MORPHEUS_ROOT}
+
+gpuci_logger "Archiving results"
+mamba pack --quiet --force --ignore-editable-packages --ignore-missing-files --n-threads ${PARALLEL_LEVEL} -n morpheus -o ${WORKSPACE_TMP}/conda_env.tar.gz
+tar cfj ${WORKSPACE_TMP}/workspace.tar.bz --exclude=".git" --exclude="models" --exclude=".cache" ./
+ls -lh ${WORKSPACE_TMP}/
+
+gpuci_logger "Pushing results to ${DISPLAY_ARTIFACT_URL}"
+aws s3 cp --no-progress "${WORKSPACE_TMP}/conda_env.tar.gz" "${ARTIFACT_URL}/conda_env.tar.gz"
+aws s3 cp --no-progress "${WORKSPACE_TMP}/workspace.tar.bz" "${ARTIFACT_URL}/workspace.tar.bz"
+
+# gpuci_logger "Running conda build for morpheus"
+# ZS=$(sccache --zero-stats)
+# CONDA_ARGS="--no-build-id --output-folder ${CONDA_BLD_DIR} --no-test" ${MORPHEUS_ROOT}/ci/conda/recipes/run_conda_build.sh morpheus
+
+# gpuci_logger "sccache usage for morpheus conda build:"
+# sccache --show-stats
+
+# gpuci_logger "Archiving conda builds"
+# tar cfj ${WORKSPACE_TMP}/conda_build.tar.bz ${CONDA_BLD_DIR}
+# ls -lh ${WORKSPACE_TMP}/
+
+# gpuci_logger "Pushing conda builds to ${DISPLAY_ARTIFACT_URL}"
+# aws s3 cp --no-progress "${WORKSPACE_TMP}/conda_build.tar.bz" "${ARTIFACT_URL}/conda_build.tar.bz"
```

Comment:
Should this debugging code be removed? If we want to leave it in there as an option, then use an environment variable to control whether or not this should be called.